src/components/look-controls.js

```diff
@@ -211,6 +232,17 @@ module.exports.Component = registerComponent('look-controls', {
   onMouseDown: function (event) {
     this.mouseDown = true;
     this.previousMouseEvent = event;
+
+    var sceneEl = this.el.sceneEl;
+    var canvasEl = sceneEl.canvas;
+
+    if (!this.pointerLocked) {
```

Comment:
if I decline the Pointer Lock permission prompt, every subsequent mouse click shouldn't try to request Pointer Lock again. if I enter Pointer Lock, press ESC to exit Pointer Lock, then it's fine to try to request Pointer Lock again.

I couldn't figure out a reliable way of figuring out if the user declined Pointer Lock permission. `{moz}pointerlockerror` I couldn't get to fire.

ideally, this API would be covered by the Permissions API, but it's not (yet).